components/rmrk/Gallery/Search/SearchSortDropdown.vue

```diff
@@ -1,6 +1,19 @@
 <template>
   <b-field>
+    <b-dropdown
+      v-if="multipleSelect"
+      multiple
+      v-model="selectedActionArray"
+      class="select-dropdown">
+      <template #trigger>
+        <b-button type="is-primary" icon-right="menu-down"> Sort by </b-button>
```

Comment:
Missing translation here + fix icon
```suggestion
        <b-button type="is-primary" icon-right="caret-down"> Sort by </b-button>
```